src/libraries/Microsoft.PowerFx.Core/Binding/INameResolver.cs

```diff
@@ -36,6 +38,8 @@ internal interface INameResolver
 
         IEnumerable<TexlFunction> Functions { get; }
 
+        ImmutableDictionary<string, GlobalSymbol> GlobalSymbols { get; }
```

Comment:
Note - this is a breaking change and means we'll need an update to PAClient.

Reply:
Yes.

Reply:
We could either:
1. Non-breaking, put it on INameResolver2 (and then QI for that) 
2. If we stay breaking, then we need a companion PR for PowerApps.